pkg/cmd/task/start.go

```diff
@@ -214,6 +214,14 @@ func parseTask(taskLocation string, p cli.Params) (*v1beta1.Task, error) {
 	if err := yaml.UnmarshalStrict(b, &task); err != nil {
 		return nil, err
 	}
+	for _, param := range task.Spec.Params {
```

Comment:
Is not better to have this common function in params package, and use it. We can then use for pipeline, task and clustertask

Reply:
@piyush-garg , yes, correct, I will write a common function in params package. `-f` option for clustertask has not been implemented yet, so I am doing this currently for pipeline and task. Or may be I can open a new PR to implement `-f` flag for clustertask and one that gets merged we proceed with this one, what do you say?